smithy-swift-codegen/src/test/kotlin/HttpProtocolUnitTestRequestGeneratorTests.kt

```diff
@@ -784,7 +780,8 @@ class HttpProtocolUnitTestRequestGeneratorTests {
                 throw serviceError
             })
         }
- """
```

Comment:
May the single space someone left in front of these three quotes die a horrible death.